<launch>
  <arg name="robot_prefix"              default="$(env MMPUG_SYSTEM_ID mt007)" />
  <arg name="sensor_index"              default="$(env DTC_SENSOR_PAYLOAD_INDEX 7)" />
  <arg name="PORT"                      default="$(env MMPUG_PORT 6000)" />
  <arg name="HOST"                      default="$(env MMPUG_IP 10.3.1.99)" />
  <arg name="JPEG_COMPRESSION_QUALITY"  default="50" />
  <!-- FPS of 15 means 2 frames per second -->
  <arg name="FPS"  default="15" />
  


  <group>
    <push-ros-namespace namespace="$(var robot_prefix)" />
        <node pkg="image_sender" exec="client" name="client" output="screen">
            <param name="sensor_index"  value="$(var sensor_index)"   type="str"/>
            <param name="robot_prefix"  value="$(var robot_prefix)"   type="str"/>
            <param name="PORT"          value="$(var PORT)"          />
            <param name="HOST"          value="$(var HOST)"      />
            <param name="JPEG_COMPRESSION_QUALITY"  value ="$(var JPEG_COMPRESSION_QUALITY)"  type="int"/>
            <param name="FPS"  value ="$(var FPS)"  type="int"/>

        </node>
  </group>
</launch>
